docs/daemon/kwild.mdx

```diff
@@ -0,0 +1,145 @@
+KWILD_CHAIN_RPC_LISTEN_ADDRESS="tcp://127.0.0.1:26656"
+
+kwild --root_dir "path/to/root/dir"
+```
+
+```bash title="Command line flags"
+kwild --root_dir "path/to/root/dir" --log.level "debug" --chain.rpc.listen_addr "tcp://127.0.0.1:26656"
+```
+
+```bash title="Both environment and command line flags"
+KWILD_LOG_LEVEL=debug
+KWILD_CHAIN_RPC_LISTEN_ADDRESS="tcp://127.0.0.1:26656"
+
+kwild --root_dir "path/to/root/dir" --log.level "info" --chain.rpc.listen_addr "tcp://192.168.1.20:30000"
+```
+If both flags and env variables are set, flags take precedence over environment variables. Therefore, in this example, the Kwil application uses `info` log level and RPC listen address as `tcp://192.168.1.20:30000`.
+
+Refer to the [translation guide](./config/settings#config-override) for more information on mapping the config parameters to the command line flags and environment variables.
+
+
+## Deploy a Kwil Network
+
+A private Kwil network with a standalone node can be deployed by running `kwild` as shown below:
+
+```bash
+kwild --root_dir "path/to/root/dir"
+```
+The root directory must have the required configuration for the node to start successfully. If you are using [quickstart mode](./kwild#quickstart), this root directory will be generated for you if it does not exist. 
+
+Whether this node starts as a validator or a non-validator is determined by its presence in the initial validator list within the genesis file.
+
+
+## Connecting To A Kwil Network
+
+To connect to an existing Kwil network during startup:
+- Obtain the `genesis.json` for the desired network from another participant, and ensure that it is placed properly in your `root_dir`.
+- Update the `chain.p2p.persistent_peers` with the peer addresses from the network using one of the following methods and start the node. `persistent_peers` is a comma-separated string of `nodeid@ip:port` of peers to establish and maintain connections upon startup.
+
+  ```bash title="Command line flag:  --chain.p2p.persistent_peers"
+  kwild --root_dir "path/to/root/dir" --chain.p2p.persistent_peers "d128266b8b9f64c313de466cf29e0a6182dba54d@172.10.100.2:26656,9440f4a8059cf7ff31454973c4f9c68de65fe526@172.10.100.3:26656"
+  ```
+
+  ```bash title="Environment variable:  KWILD_CHAIN_P2P_PERSISTENT_PEERS"
+  KWILD_CHAIN_P2P_PERSISTENT_PEERS="d128266b8b9f64c313de466cf29e0a6182dba54d@172.10.100.2:26656,9440f4a8059cf7ff31454973c4f9c68de65fe526@172.10.100.3:26656"
+
+  kwild --root_dir "path/to/root/dir"
+  ```    
+
+  ```bash title="persistent_peers option in config.toml"
+  [chain.p2p]
+  persistent_peers = "d128266b8b9f64c313de466cf29e0a6182dba54d@172.10.100.2:26656,9440f4a8059cf7ff31454973c4f9c68de65fe526@172.10.100.3:26656"
+
+  kwild --root_dir "path/to/root/dir"
+  ```
+
+4. Note that, this node joins the network by default as a non-validator. Please refer to the [Validator guide](./validator#federated-consensus-based-join-process) for more information on becoming a validator node.
```

Comment:
small nit, but can we either:
1. name it "validator guide"
2. name it "Validator Guide"

just for consistency